tests/integration/playwright/playwright.config.ts

```diff
@@ -30,7 +30,7 @@ const config: PlaywrightTestConfig = {
   /* Opt out of parallel tests on CI. */
   workers: process.env.CI ? 1 : undefined,
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: 'html',
+  reporter: [["html", { open: "never" }]],
```

Comment:
This is by itself a very nice find!

Reply:
Thank you ! It was very annoying and in parallel runs locally, I could not have the tests continue running after the browser windows opened and I closed it. So reading mode into playwright doc... and found this ! 😄